sdk/tracing/azure-core-tracing-opencensus/README.md

```diff
@@ -0,0 +1,128 @@
+
+## Examples
+The following sections provides examples of using tracing-opencensus plugin with some of the Azure Java sdk libraries:
+### Using the plugin package with Http Client libraries
+- Sync create a Secret using [azure-keyvault-secrets][azure-keyvault-secrets] with tracing enabled.
+    
+    Users can additionally pass the value of the current tracing span to the sdk's using key **"opencensus-span"** on the [Context][context] object as shown below:
+      and value of current tracing span.
+	```java
+	try (Scope scope = tracer.spanBuilder("tracing-user-parent").startScopedSpan()) { 
+		secretClient.setSecretWithResponse(new Secret("BankAccountPassword", "f4G34fMh8v"),
+	    		new Context("opencensus-span", tracer.getCurrentSpan()));
+	}
+	```
+- Async Create a Secret using [azure-keyvault-secrets][azure-keyvault-secrets] with tracing enabled
+    
+    Users can additionally pass the value of the current tracing span to the sdk's using key **"opencensus-span"** on the subscriberContext of the calling method:
+	```java
+	try (Scope scope = tracer.spanBuilder("tracing-user-parent").startScopedSpan()){
+	    Secret secretResponse = secretAsyncClient.setSecret(new Secret("BankAccountPassword", "f4G34fMh8v")
+						.expires(OffsetDateTime.now().plusYears(1)))
+						.subscriberContext(Context.of("opencensus-span", tracer.getCurrentSpan()))
+						.block();
+	}
+	```
+
+### Using the plugin package with AMQP Client libraries
+- Sync send single event using [azure-messaging-eventhubs][] with tracing
+    
+    Users can additionally pass the value of the current tracing span to the EventData object with key **"opencensus-span"** on the [Context][context] object:
+	```java
+	try (Scope scope = tracer.spanBuilder("tracing-amqp").startScopedSpan()) {
+		// Create an event to send.
+		final EventData eventData = new EventData("Hello world!".getBytes(UTF_8));
+		eventData.context(new Context("opencensus-span", tracer.getCurrentSpan()));
+		producer.send(eventData).doOnSuccess((ignored) -> System.out.println("Event sent.")).block();
+	}
+	```
+## Troubleshooting
+
+## Next steps
+Several Java SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Tracing:
+
+#### Hello World Samples
+* [HelloWorld.java][sample_helloWorld] - and [HelloWorldAsync.java][sample_helloWorldAsync] - Contains samples with tracing enabled for following scenarios:
+    * Create a queue client using [azure-storage-queue][azure-storage-queue] Java sdk
+    * Enqueue and dequeue messages using the created QueueClient
+#### List Operations Samples
+* [ListOperations.java][sample_list] and [ListOperationsAsync.java][sample_list_async] - Contains samples with tracing enabled for following scenarios:
+    * Create a Key
```

Comment:
For these, use title casing. And end them with periods, similar to the ones below. Line 74 and 93 is also missing a period.